resources/views/visit-transfer/site/application/referees.blade.php

```diff
@@ -34,37 +34,39 @@
                             {!! Form::open(["route" => ["visiting.application.referees.post", $application->public_id], "method" => "POST"]) !!}
 
                             <div class="col-md-6">
-                                {!! ControlGroup::generate(
-                                    Form::label("referee_cid","Referee CID"),
-                                    Form::text("referee_cid"),
-                                    Form::help("Please ensure this is correct.")
-                                )->withAttributes(["id" => "refereeCidHelp"]) !!}
+                                <div id="refereeCidHelp">
```

Comment:
Where we are keeping the control group divs, we need the div to use `class="form-group"` to ensure proper styling in bootstrap

Reply:
I tried that and it totally messed the styling up. It looks fine in browser without - can you test a few of them please?

Reply:
Fair enough, they look fine without them (although I don't really see a difference with them), so I would just suggest we leave them out for now.